vesc_driver/src/vesc_packet.cpp

```diff
@@ -34,10 +34,12 @@
 #include <iterator>
 #include <memory>
 #include <string>
+#include <cmath>
```

Comment:
I may be missing it but I don't see any functions from `cmath` being used below. Was there a reason it was added?

Reply:
from commit 654ba11